jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -198,7 +198,7 @@ YUI.add('user-profile', function() {
           model.owner = model.ownerTag;
           return model;
         });
-      } else {
+      } else if (data.map) {
```

Comment:
I was "meh" about how to do this. In the end, I decided to check for the function that was actually used in the subsequent block. I could go either way.